### PR TITLE
chore: Remove Meteor.Streamer client usage

### PR DESCRIPTION
**Pull request by ggazzo**
[ARCH-1112]
the idea is to remove the old meteor-streamer package, and use the same interface that the new sdk uses.

We are not using the new sdk yet, we are using a wrapper implementing the same methods but based on meteor.

that way we can refactor all the code without worrying about  new bugs being introduced, then we can run our standalone application

## Further comments

[ARCH-1112]: https://rocketchat.atlassian.net/browse/ARCH-1112?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/meteor/client/views/meet/CallPage.tsx [hunk 1]
```diff
@@ -71,36 +71,34 @@ const CallPage: FC<CallPageProps> = ({
...
-				visitorId,
-			);
-			Notifications.onRoom(roomId, 'webrtc', (type: any, data: any) => {
+			Notifications.onVisitor(WEB_RTC_EVENTS.WEB_RTC, visitorId, (type: any, data: any) => {
+				if (data.room == null) {
+					return;
+				}
+				webrtcInstance.onUserStream(type, data);
+			});
+			// TODO: I have no idea what this is doing, how this works, why it's here or how to type it
```
Comment (sampaiodiego) [inline apps/meteor/client/views/meet/CallPage.tsx:80]: once this is merged nobody will know who "I" is.. maybe you should sign this comment?

btw, I don't see much relevance of having this comment in the code as well

File: apps/meteor/ee/app/canned-responses/client/streamer.js
Comment (sampaiodiego) [inline apps/meteor/ee/app/canned-responses/client/streamer.js:1]: delete file?

Comment (ggazzo) [reply apps/meteor/ee/app/canned-responses/client/streamer.js:1]: my bad

File: apps/meteor/client/views/meet/CallPage.tsx (hunk 1 above)
Comment (ggazzo) [reply apps/meteor/client/views/meet/CallPage.tsx]: just wanted to make clear why I inserted any instead o typing the stream. I

File: apps/meteor/ee/app/canned-responses/client/streamer.js
Comment (ggazzo) [reply apps/meteor/ee/app/canned-responses/client/streamer.js:1]: my bad thanks

**Review by sampaiodiego — APPROVED** [review]

**Pull request merged**